example/app/src/main/java/com/tvd12/ezyfoxserver/client/example/extension/Extensions.kt

```diff
@@ -0,0 +1,20 @@
+package com.tvd12.ezyfoxserver.client.example.extension
```

Comment:
The file name should be `EditTextExtensions` or `UiExtensions` because in the future, we maybe have a lot of extension methods then write all in the one file is not a good idea, how do you think?

Reply:
I totally agree